ee/modules/030-cloud-provider-vsphere/template_tests/module_test.go

```diff
@@ -365,13 +365,10 @@ vcenter:
     server: "myhost"
     datacenters:
       - "X1"
-    externalNetworkNames:
-      - aaa
-      - bbb
-    internalNetworkNames:
-      - ccc
-      - ddd
-    vmFolderPath: dev/test
+
+nodes:
+  externalVmNetworkName: aaa,bbb
+  internalVmNetworkName: ccc,ddd
```

Comment:
What's wrong with lists? Why do we want it concatenated in a string?

Reply:
They were lists but now they are strings, concatenated with a ','
Don't know the details honestly. It's terraform or something. We had the condition inside the template and test was made only for kubernetes < 1.22
Now the condition was removed, that's why I have changed the test